delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1041,21 +1040,32 @@ def updateServiceInDB(service_id_in_db, devApiID, region, statusval) {
 def loadEventsModule() {
 }
 
+	
 def loadServiceInfo() {
 	def token = setCredentials()
 	def url = "$g_base_url/platform/services?domain=$domain&service=$service&isAdmin=true"
 	def curlOutput = sh (script: "curl -H \"Content-Type: application/json\" \
-			-H \"Authorization: $token\" \"$url\" | $jq -r '.data[0] | [.created_by] | join(\",\")'", returnStdout: true)
-	def values = curlOutput.split(',')
-
+			-H \"Authorization: $token\" \"$url\"", returnStdout: true)
+  	echo "CURL o/p: $curlOutput"
+  	
+	def jsonParser = new groovy.json.JsonSlurper()
+	def resultJson = jsonParser.parseText(curlOutput)
+  	echo "Resultjson: $resultJson"
+	def createby = ''
+  	def pcreated = resultJson.data[0].created_by
```

Comment:
what is the difference between this line and 1058? the createdby variable captures the same value, we can delete this line.

Reply:
Was an unneeded code line 1055 . Removed